Makefile

```diff
@@ -131,29 +146,35 @@ KUSTOMIZE ?= $(LOCALBIN)/kustomize
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint
+KIND ?= $(LOCALBIN)/kind
+GINKGO ?= $(LOCALBIN)/ginkgo
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.8.7
 CONTROLLER_TOOLS_VERSION ?= v0.9.0
 
 KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
-.PHONY: kustomize
```

Comment:
Did you remove these because they reference actual files?

Reply:
oops, that wasn't meant to be removed :sweat_smile:

Reply:
Should be fixed with commit https://github.com/operator-framework/oria-operator/pull/69/commits/9cfafd2158ea14da68bb62915c3defd4878b3ba2